Sources/HummingbirdWSCore/WebSocketStateMachine.swift

```diff
@@ -69,7 +69,7 @@ struct WebSocketStateMachine {
             let code: WebSocketErrorCode = if dataSize == 0 || closeCode != nil {
                 // codes 3000 - 3999 are reserved for use by libraries, frameworks, and applications
                 // so are considered valid
-                if case .unknown(let code) = closeCode, code < 3000 || code > 3999 {
+                if case .unknown(let code) = closeCode, code < 3000 {
```

Comment:
What's the change for? Is the comment wrong?

Reply:
I need to update the comment

Reply:
Looking at the RFC should probably have changed that to 4999 instead of removing it. See https://www.rfc-editor.org/rfc/rfc6455.html#section-7.4.2

Reply:
Updated comment and check